ansible/host_vars/balaka.openmrs.org/vars

```diff
@@ -31,17 +31,25 @@ docker_deployment:
  - qa-refapp
  - modules-refapp
  - uat-refapp
+ - emr-3-dev
 
 docker_deployable_images:
  - repository: openmrs/openmrs-reference-application-distro
    compose: demo,qa-refapp,modules-refapp,uat-refapp
+ - repository: openmrs/openmrs-reference-application-3-gateway
```

Comment:
This is sort of not used anymore, I either use `ssh` from bamboo/ci or I just run ansible when I need a new image. 

Now, I don't think this machine is capable to handling another docker-compose, I think it's already on its limit from datadog, no?

Reply:
A few other machines to consider I reckon: dowa, namanga. 

But here's what we have today: https://app.datadoghq.com/dashboard/ee2-evy-23q/machine-resources?tpl_var_host=balaka.openmrs.org&from_ts=1628426879292&to_ts=1628599679292&live=true

Reply:
Yeah, it is a bit busy. My thought process here is that this will eventually become the new demo site and the only other available machine at the same prod tier is mua (i.e., the website). Not sure the best way forward?

Reply:
> This is sort of not used anymore, I either use ssh from bamboo/ci or I just run ansible when I need a new image.

Ok, that makes me feel better because I was trying to figure out how this connecting to anything (and I've had less than ideal experience with Dockerhub hooks in the past)!